lib/ansible/plugins/loader.py

```diff
@@ -28,6 +27,12 @@
 from ansible.utils.display import Display
 from ansible.utils.plugin_docs import add_fragments
 
+try:
```

Comment:
I think all of this stuff should go into a separate compat module abstracting hacks away.